react.component/src/context/Context.test.tsx

```diff
@@ -0,0 +1,106 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { describe, expect, test, beforeAll, afterEach, afterAll } from 'vitest';
+import { render, screen, waitFor, act, fireEvent } from '@testing-library/react';
+import { App } from '../app/App';
+import { ErrorPage } from '../errorPage';
+import { createMemoryRouter, RouterProvider } from 'react-router-dom';
+
+const fakeInfo = {
+  birth: 'fakebirth',
+  race: 'fakeRace',
+  death: 'fakeDath',
+  hair: 'fakeHair',
+  gender: 'fakeGender',
+  height: 'fakeHeight',
+  realm: 'fakeRealm',
+  spouse: 'fakeSpouse',
+  wikiUrl: 'fakewikiurl',
+  _id: 'fakeID'
+}
+
+const fakeCard1 = {
+  key: 'fakeKey1',
+  name: 'fakeName1',
+  ...fakeInfo,
+};
+const fakeCard2 = {
+  key: 'fakeKey2',
+  name: 'fakeName2',
+  ...fakeInfo,
+};
+const fakeCard3 = {
+  key: 'fakeKey3',
+  name: 'fakeName3',
+  ...fakeInfo,
+};
```

Comment:
make sure to annotate those mocks as the same types that you've used to annotate response api in the application itself

Reply:
![image](https://user-images.githubusercontent.com/88984698/230085249-76c80b86-f938-4cf4-924c-9c34e830a96a.png)

is it correct?